### PR TITLE
feat: landing page

### PR DESCRIPTION
**Pull request by harshitadaswani**
landing page includes navigation, banner, categories and footer

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,35 +9,35 @@
...
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+        integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
 </head>
+
 <body>
 
     <nav class="navigation">
         <div class="nav-left">
-
             <a href="index.html" class="txt-l fw-black p-s m-s">Crypto&nbsp;Store</a>
```
Comment (barbaricankit) [inline index.html:20]: what does fw-black means? use full names instead of p and m.

File: index.html [hunk 2]
```diff
@@ -8,35 +9,35 @@
...
-
         </div>
-        <div class="searchbar-input">
 
+        <div class="searchbar-input">
             <span class="icon-input">
                 <i class="fas fa-search icons"></i> Search items here
             </span>
 
             <input type="search" name="searchbar" class="p-s m-s input-box br-s" placeholder="">
```
Comment (barbaricankit) [inline index.html]: give a value to your placeholder, use full names here too for all the properties

File: index.html [hunk 3]
```diff
@@ -8,35 +9,35 @@
...
-
         </div>
-        <div class="nav-right">
 
+        <div class="nav-right">
             <button class="btn btn-link m-m">
```
Comment (barbaricankit) [inline index.html:32]: use margin-med instead

File: index.html [hunk 4]
```diff
@@ -8,35 +9,35 @@
...
 
             <a href="/" class="txt-m fw-black p-s m-s">
                 <div class="badge">
                     <i class="fas fa-heart icons"></i>
                     <div class="number-badge txt-s">10</div>
-                </div> 
+                </div>
             </a>
 
             <a href="/" class="txt-m fw-black p-s m-s">
```
Comment (barbaricankit) [inline index.html:43]: either use sm or small, avoid using single-letter names

File: index.html [hunk 5]
```diff
@@ -45,14 +46,105 @@
...
-            
         </div>
-
     </nav>
 
     <main>
-        
+        <section class="crypto-flex">
+            <div class="img-container">
+                <img src="./assets/image1.png" alt="">
```
Comment (barbaricankit) [inline index.html]: provide a value to alt

File: index.html [hunk 6]
```diff
@@ -45,14 +46,105 @@
...
+
+        <section>
+            <h2 class="content-fluid p-s">Every Industry can benefit from NFT</h2>
+
+            <p class="p-s content">Our ready-made NFT platform is the place where art becomes unique for traders and
+                creators. It brings together collectors and creators, allowing them.</p>
+
+            <div class="category-container">
+                <div class="category p-l m-l br-m">
+                    <a href="" class="p-s m-s">
```
Comment (barbaricankit) [inline index.html]: give a value to href

File: stylesheets/index.css [hunk 7]
```diff
@@ -12,3 +12,100 @@
...
+.content {
+    width: 60%;
+    margin: auto;
+}
+
+.content-fluid {
+    width: 50%;
+    margin: auto;
+}
+
```
Comment (barbaricankit) [inline stylesheets/index.css]: both are having same properties so you could have a class name margin-auto and used that in both of them

**Review by barbaricankit — COMMENTED** [review]
I loved that you have used css variables for colors.
You are using very short class names in some places which you can change to more understandable words. You also have some attributes in html tags where you are not providing any values, please check those.
For common properties try to have one classname and use that everywhere.

File: stylesheets/index.css (hunk 7 above)
Comment (harshitadaswani) [reply stylesheets/index.css]: done

File: index.html (hunk 6 above)
Comment (harshitadaswani) [reply index.html]: done

File: index.html (hunk 5 above)
Comment (harshitadaswani) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (harshitadaswani) [reply index.html]: placeholder value done

Comment (harshitadaswani) [reply index.html]: will keep in mind the full names during the next project

File: index.html (hunk 1 above)
Comment (harshitadaswani) [reply index.html:20]: will keep in mind during the next project

File: index.html (hunk 3 above)
Comment (harshitadaswani) [reply index.html:32]: will keep in mind during the next project

File: index.html (hunk 4 above)
Comment (harshitadaswani) [reply index.html:43]: will keep in mind during the next project

**Review by SuyashTiwariCoder — APPROVED** [review]
It's Looking fine , You can merge

**Review by ozairkhan2810 — APPROVED** [review]
looks good, can merge

**Review by rkapoor10 — APPROVED** [review]
Looks good to merge.

**Pull request merged**